source/packages/services/installer/src/commands/modules/service/certificateVendor.ts

```diff
@@ -51,16 +55,322 @@ export class CertificateVendorInstaller implements ServiceModule {
         this.assetLibraryStackName = `cdf-assetlibrary-${environment}`;
         this.commandAndControlStackName = `cdf-commandandcontrol-${environment}`;
     }
+    
+    if (updatedAnswers.certificateVendor === undefined) {
```

Comment:
Something went wrong in updating this file. This if statement now lives outside of any method. It looks like it belongs in the prompts function which has been removed.

Reply:
will fix